javascript-loops.tar/javascript-loops/main.js

```diff
@@ -0,0 +1,75 @@
+/* exported getNumbersToTen,
+            getEvenNumbersToTwenty,
+            repeatWord,
+            logEachCharacter,
+            doubleAll,
+            getKeys,
+            getValues
+ */
+function getNumbersToTen() {
+  const numbers = [];
+  let currentNumber = 1;
+  while (currentNumber <= 10) {
+    numbers.push(currentNumber);
+    currentNumber++;
+  }
+  return numbers;
+}
+console.log(getNumbersToTen());
+
+function getEvenNumbersToTwenty() {
+  const evenNumbers = [];
+  let currentNumber = 2;
+  while (currentNumber <= 20) {
+    evenNumbers.push(currentNumber);
+    currentNumber += 2;
+  }
+  return evenNumbers;
+}
+console.log(getEvenNumbersToTwenty());
+
+function repeatWord(word, times) {
+  let count = 1;
+  let repeated = '';
+  while (times > count) {
+    repeated += word;
+    count++;
+  }
+  return repeated;
+}
+console.log(typeof repeatWord('cool', 10));
+console.log(repeatWord('cool', 10));
+
+function logEachCharacter(string) {
+  for (let i = 0; i < 3; i++) {
```

Comment:
`i < 3` is not right. That would only work for strings that are 3 characters long. What if I passed in a string longer than three characters? How could I re-write that expression to dynamically handle any length of string that is passed in from the argument?